board/config.h

```diff
@@ -7,11 +7,17 @@
 //#define DEBUG_SPI
 //#define DEBUG_FAULTS
 
-#ifdef STM32F4
+#ifdef STM32H7
+  #define PANDA
+  #include "stm32h7xx.h"
+  #define FREQ 550U // 550Mhz
```

Comment:
Let's rename this to `CORE_FREQ` or something, might be easier to understand when you come across it. Also, `Hz` not `hz` ;)